File: 1013 - The Greatest/Main.c
#include <stdio.h>

int main()
{
    int a, b, c;
    scanf("%d %d %d", &a, &b, &c);
    if (a > b)
    {
        b = a;
    }
    printf("%d eh o maior\n", c > b ? c : b);
    return 0;
}

File: 1013 - The Greatest/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int a = scan.nextInt();
        int b = scan.nextInt();
        int c = scan.nextInt();
        if (a > b) {
            b = a;
        }
        System.out.printf("%d eh o maior\n", c > b ? c : b);
        scan.close();
    }
}